problems/reverseString.yaml

```
@@ -0,0 +1,27 @@
+id: 4
+title: ReverseString
+text: |
+  Write a function called reverseString that takes a string parameter 
+  and returns the reverse version of said string.
+  
+  Refer to example below.
+  
+  reverseString('YellowBrickCode') = edoCkcirBwolleY
+  reverseString('Google') = elgooG
+code: |
+  function reverseString(str) {
+    return str;
+  }
+
+  assert(reverseString('hello') === olleh);
```

the outputs should also be in single quotes for these assertions to pass - maybe also change in examples on lines 9 and 10
```suggestion
  assert(reverseString('hello') === 'olleh');
```